tslint.json

```diff
@@ -2,6 +2,11 @@
   "rulesDirectory": [
     "node_modules/codelyzer"
   ],
+  "linterOptions": {
+    "exclude": [
+        "events.json"
```

Comment:
Instead of the disable-tslint in the json file try pointing to the file directly.

src/app/cube/about-us/components/timeline/events.json